doc/source/custom_directives.py

```diff
@@ -751,17 +751,17 @@ def render_library_examples(config: pathlib.Path = None) -> bs4.BeautifulSoup:
         # Group the examples by the ExampleConfig.groupby value:
         examples = defaultdict(list)
         example_config = ExampleConfig(config, app.srcdir)
-        for example in example_config:
+        for _example in example_config:
```

Comment:
why is this change?

Reply:
Ops, I misread that. Let me make some corrections.